dag_files/dag_energy.yml

```diff
@@ -90,24 +90,26 @@ steps:
   data://garden/ember/2022-08-01/combined_electricity_review:
     - data://garden/ember/2022-08-01/european_electricity_review
     - data://garden/ember/2022-08-01/global_electricity_review
+  data://garden/ember/2022-12-13/combined_electricity:
+    - data://garden/ember/2022-08-01/european_electricity_review
+    - data://garden/ember/2022-12-13/yearly_electricity
```

Comment:
Yes, this is understandably confusing. We used to use Ember's Global Electricity Review as well as the European Electricity Review, but then Ember released a Yearly Electricity Data, which was more complete than the Global Review, but, for some reason, does not include as much data for European countries as the European Review.
That's why I created the Combined Electricity Review. But on the last update, I realised it's better just to call it "Combined Electricity".
I have renamed things to stop mentioning the global review. However, I left some instances of "global" to distinguish global data from European data.

Reply:
Makes sense.

How is the dataset `data://garden/ember/2022-08-01/global_electricity_review` cooked, though?

Reply:
That dataset is not in the dag anymore, I've removed it. Lars suggested we should not remove datasets, but archive them (so that we can still know how to recreate all dataset). For that, I think we should create a `dag_archived`, but that's not yet implemented. And I thought keeping these things in the dag would be more confusing. Let me know if you disagree.

Reply:
Gotcha, you deleted these!